README.md

```diff
@@ -6,15 +6,15 @@ https://stratisplatform.com
 Bitcoin Implementation in C#
 ----------------------------
 
-Stratis is an implementation of the Bitcoin protocol in C# on the [.NET Core](https://dotnet.github.io/) platform.  
-The daemon is a full implementation of the Stratis Bitcoin Full Node.  
```

Comment:
I have a plugin on my text editor that removes trailing spaces. Generally, this is positive, except for when it adds complexity to a review...